tests/images/polar_alignment_utils_new.py

```diff
@@ -0,0 +1,60 @@
+from PIL import Image, ImageOps
+
+
+def fix_image_size(max_width, max_height, image):
+    """Resizes images while maintaining the same aspect ratio.
+
+    Args:
+        max_width: Image width
+        max_height: Image height
+        image: Image to be resized
+
+    Returns:
+        The resized image.
+    """
+    width_ratio = max_width / image.size[0]  # resized image ratios
+    height_ratio = max_height / image.size[1]
+
+    new_width = int(width_ratio * image.size[0])  # proportional width, height value calculations
+    new_height = int(height_ratio * image.size[1])
+
+    new_image = image.resize((new_width, new_height))
+    return new_image
+
+
+def grid_display(
+    background_jpg,
+    overlay_png,
+    width=4800,
+    height=2400,
+    alpha=0.2,
```

Comment:
So this makes me think that the image overlay is not quite doing what we want. Can you upload a picture of the result on the main PR page?

The PNG should be transparent (that is, it has an "alpha channel") except for the grid mark lines, so when you overlay the two images, it should automatically blend with the background without needing to set the alpha. If you set this alpha, it is going to make the grid lines themselves mostly transparent.  I think you mentioned you were having a hard time seeing them so maybe it was that.

JPGs don't have alpha channels, so it won't work with a jpg.  The issue might be that you would have to first convert the jpg, then add the overlay on top.

It looks like you want to be using the `Image.paste` command, not the `Image.blend` (although `paste` is a silly name).

https://stackoverflow.com/questions/5324647/how-to-merge-a-transparent-png-image-with-another-image-using-pil (although beware this is a very old answer)